internal/pkg/fetch/fetch.go

```diff
@@ -37,7 +37,7 @@ import (
 )
 
 func Fetch(sbomURL string, opts *options.FetchOptions) error {
-	document, err := fetchDocument(sbomURL, opts)
+	document, err := document(sbomURL, opts)
```

Comment:
Could this cause issues since the variable has the same name as a function in the same namespace?